### ISSUE TITLE
area2point.fix_geotiff_locations unexpectedly sets noDataValue=0

### ISSUE THREAD

**Issue by asjohnston-asf** [ASFHyP3/hyp3-lib#232]
This function is used by GAMMA RTC to translate all output rasters from pixel as area to pixel as point after RTC processing is completed: https://github.com/ASFHyP3/hyp3-gamma/blob/develop/hyp3_gamma/rtc/rtc_sentinel.py#L342

`fix_geotiff_locations()` overwrites any existing noDataValue and sets it to 0: https://github.com/ASFHyP3/hyp3-lib/blob/develop/hyp3lib/area2point.py#L33 .

This is a particular issue for DEM output files, which may have valid pixel values of zeros.  For example, for an RTC image over death valley (S1A_IW_GRDH_1SDV_20201002T135151_20201002T135216_034618_0407ED_A3FB), the DEM packaged with the output has a clear ring of no data pixels where the terrain transitions from above sea level to below sea level.  `fix_geotiff_locations()` should be modified to only convert from pixel as area to pixel as point, and any noDataValue business logic should reside elsewhere.

![Screenshot from 2021-01-14 17-02-26](https://user-images.githubusercontent.com/17994518/104672100-94b0f500-568b-11eb-9267-1e63683e77c9.png)

**Comment by rudigens** [comment]
Very good call, Andrew. I have thought about this issue for a while but could not come up with an efficient way how to fix it. Might want to have another discussion with everybody around about what our options are. The cleanest way would actually to fix this in the DEM tiles but it is a lot of work and not that trivial to get it right at the coastlines.

**Comment by talogan** [comment]
I agree this is an issue wherever we have land below sea level (ocean
coasts).  However, I'm not sure that we can fix this.

Gamma's nodata value is 0.  Wherever a DEM has a zero value, no terrain
correction is performed and, instead, output pixels are set to zero.  Thus
if we have DEM values less than zero, we'll get a ring of nodata values at
the transition from positive to negative as is evidenced in the Death
Valley scene that Andrew processed.
